token-utility-whitepaper.md

```diff
@@ -1 +1,163 @@
+Although OBX is not deflationary, token holders are incentivised to stake as Aggregator node operators or as staking delegates by means of a reward. This will lock up OBX in staking pools and reduce the number of OBX in circulation. 
+
+In summary, OBX has four utilities:
+1.	To provide a reward to incentivise node operators  to correctly operate Obscuro nodes and provide a reliable service.
+1.	A medium in which users can pay fees and node operators can pay their stake and receive rewards to cover operational costs.
+1.	A means to vote on proposals for changes to Obscuro via a governance model.
+1.	A means to fund and incentivise the ongoing development of the Obscuro ecosystem via development opportunities, grants and competitions.
+
+## Token Issuance and Rewards
+_This section is non-normative_
+
+OBX is issued by the Obscuro Foundation. The Token Generation Event (TGE) occurs when the Obscuro Testnet is substantially complete and application builders and Obscuro participants can meaningfully interact with the Testnet. The Obscuro DAO controls the ongoing tokenomics for Obscuro Mainnet according to the DAO’s constitution.
+The initial token supply is 1,000,000,000 OBX. There is no minting or burning in the Obscuro protocol design. At a future date the Obscuro DAO will have the option of voting to issue more tokens if required.
+The allocations aim to reward ongoing participation in Obscuro and minimise short-termism. The largest allocations with unlocking from the get-go are for community members, third-party integrators and application developers building on top of Obscuro so they can grow and succeed alongside Obscuro.
+An airdrop has been intentionally omitted from the allocation because typically they do not benefit projects in the medium to long term.
+
+There is an occasion where tokens are slashed although this is expected to be very rare. Specifically, Aggregator operators that use a hacked enclave on their node in an attempt to break the ledger’s integrity will be discovered by the protocol. Their punishment is to have their stake slashed and returned to the Ecosystem Fund.
+
+| RECIPIENT             | TOKEN AMOUNT   | ALLOCATION |
+|-----------------------|----------------|------------|
+| Contributor Whitelist | 20,000,000     | 2%         |
+| Public Sale           | 26,000,000     | 2.6%       |
+| Foundation Fund       | 238,000,000    | 23.8%      |
+| Ecosystem Fund        | 192,000,000    | 19.2%      |
+| Pre-Seed Investor*    | 230,000,000    | 23%        |
+| Seed Investors        | 90,000,000     | 9%         |
+| Core Team             | 180,000,000    | 18%        |
+| Service Providers     | 24,000,000     | 2.4%       |
+
+* Pre-seed investor has rights to 23% of token allocation at each token issuance.
```

Comment:
But there is a limit on this I believe? Might want to state it as appears indefinite.

Reply:
Each and every series of tokens issued over the lifetime of the Obscuro protocol are within scope of the pre-seed Warrant.